src/main/java/org/folio/services/validator/registry/ValidatorRegistryServiceImpl.java

```diff
@@ -141,19 +136,21 @@ public ValidatorRegistryService updateTenantRule(String tenantId, JsonObject val
    */
   @Override
   public ValidatorRegistryService getTenantRuleByRuleId(String tenantId, String ruleId, Handler<AsyncResult<JsonObject>> asyncResultHandler) {
+    Promise<Rule> promise = Promise.promise();
+    PostgresClient.getInstance(vertx, tenantId).getById(VALIDATION_RULES_TABLE_NAME, ruleId, Rule.class, promise);
+
     try {
-      Criteria idCrit = constructCriteria(RULE_ID_JSONB_FIELD, ruleId);
-      PostgresClient.getInstance(vertx, tenantId).get(VALIDATION_RULES_TABLE_NAME, Rule.class, new Criterion(idCrit), true, false, getReply -> {
+      PostgresClient.getInstance(vertx, tenantId).getById(VALIDATION_RULES_TABLE_NAME, ruleId, Rule.class, getReply -> {
```

Comment:
Can you remove RULE_ID_JSONB_FIELD that is no longer used?

Reply:
Removed, thank you